Cargo.toml

```diff
@@ -66,3 +67,6 @@ predicates = "1.0.2"
 
 [features]
 vendored-openssl = ['openssl/vendored']
+
+[profile.release]
+debug = true
```

Comment:
what's  this mean?

Reply:
oh i was using this to debug, great great catch.

background: wranglerjs doesn't download a binary if you're in debug mode so i was building in release mode but still wanted some sorta debug annotation i think? honestly i added this so long ago i kinda forget why i added it

Reply:
okay i thought it smelled funny 😀